lib/node/NodeMainTemplate.runtime.js

```diff
@@ -18,7 +18,7 @@ module.exports = function() {
 		return Promise.resolve(update);
 	}
 
-	function hotDisposeChunk(chunkId) {
+	function hotDisposeChunk(chunkId) { //eslint-disable-line no-unused-vars
```

Comment:
This eslint comment can be removed because you use `{"args": false}` in the rules, right?